coap/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+version: "4.3.0"
+description: libcoap library
+url: https://github.com/espressif/idf-extra-components/tree/master/coap
+dependencies:
+  idf: ">=4.2"

```

Comment:
Version `4.3.0` was supported in IDF v4.4 and on-wards. Please double check once.